lib/Helper/TemporaryFileHelperTrait.php

```diff
@@ -75,6 +75,7 @@ protected static function getTemporaryFileFromStream($stream, bool $keep = false
         fclose($dest);
 
         if (!$keep) {
+            LongRunningHelper::addTmpFilePath($tmpFilePath);
```

Comment:
Isn't that dangerous as the file still might be needed after calling `collectGarbage`? 
At least it is a potential change in behavior I would say.

Reply:
Maybe we should make `deleteTemporaryFiles()` public and do not call it implicitly when calling `cleanup()`. 
Then we don't have to deal with unexpected behavior and it's up to the dev whether to clean tmp files or not (the possibility to do so, makes definitely sense, because currently it's not possible at all).

Reply:
@BlackbitNeueMedien can you please update the PR accordingly? Thanks in advance 😊